Cargo.toml

```diff
@@ -75,6 +75,7 @@ openid = { version = "0.9.3", optional = true }
 bcrypt = "0.10.1"
 chrono = { version = "0.4.19", features = ["serde"], optional = true }
 either = "1.6.1"
+async-std = "1.9.0"
```

Comment:
I think this is going to require `features = ["tokio1"]` so that we aren't running another runtime alongside Tokio